core/embed/rust/src/ui/model_tr/component/share_words.rs

```diff
@@ -212,27 +167,18 @@ where
 {
     fn trace(&self, t: &mut dyn crate::trace::Tracer) {
         t.component("ShareWords");
-        let content = if self.is_entry_page() {
-            build_string!(
-                100,
-                self.title.inner().get_text(),
-                "\n",
-                &self.get_first_text()
-            )
-        } else if self.is_second_page() {
-            self.get_second_text()
-        } else if self.is_final_page() {
+        let content = if self.is_final_page() {
             self.get_final_text()
         } else {
-            let mut content = String::<100>::new();
+            let mut content = String::<50>::new();
             for i in 0..WORDS_PER_PAGE {
                 let index = self.word_index() + i;
                 if index >= self.share_words.len() {
                     break;
                 }
                 let word = &self.share_words[index];
                 let current_line =
-                    build_string!(20, inttostr!(index as u8 + 1), " ", word.as_ref(), "\n");
+                    build_string!(20, inttostr!(index as u8 + 1), ". ", word.as_ref(), "\n");
```

Comment:
please also use size 50
(we need a separate review of string sizes used. maybe `build_string!` should just not take this as an argument)

Reply:
Done in [d4142d4](https://github.com/trezor/trezor-firmware/pull/3135/commits/d4142d4e9aec4987e54eccb2554311810500d57f)